src/modules/chat-room/chat-room.service.ts

```diff
@@ -27,6 +28,8 @@ export class ChatRoomService {
     private readonly usersRepository: Repository<Users>,
   ) {}
 
+  private saltRounds = 5;
```

Comment:
I think it make sense have it out of class, this value does not belong to specific class. make it constant and make all letters uppercase